packages/dd-trace/src/util.js

```diff
@@ -10,7 +10,20 @@ function isFalse (str) {
   return str === 'false' || str === '0'
 }
 
+// from https://github.com/facebook/jest/blob/d1882f2e6033186bd310240add41ffe50c2a9259/packages/expect/src/utils.ts#L350
+function isError (value) {
```

Comment:
This function will get called a lot, and `toString()` could be doing a lot of work for some objects.

Reply:
got it. Changed it.